servo-tidy.toml

```diff
@@ -1,10 +1,11 @@
 [configs]
 skip-check-length = false
 skip-check-licenses = false
+check-alphabetical-order = false
 
 [ignore]
 # Ignored packages with duplicated versions
-packages = []
+packages = ["gdi32-sys", "winapi", "gl_generator", "log", "core-foundation", "core-foundation-sys", "yaml-rust", "user32-sys", "lazy_static", "core-graphics"]
```

Comment:
Would it be possible to put them one by line like in servo so it's easier to read ? :)